src/python/serenity_types/pricing/derivatives/rates/yield_curve.py

```diff
@@ -168,9 +174,9 @@ class YieldCurveAvailability(BaseModel):
     Description of the particular yield curve parameters that are available to load.
     """
 
-    build_times: List[datetime]
+    as_of_times: List[datetime]
```

Comment:
same here, as_of_time is more meaningful?

Reply:
OK